src/main/java/co/omise/models/SearchResult.java

```diff
@@ -73,6 +73,8 @@ public void setTotalPages(int totalPages) {
         private String query;
         private Map<String, String> filters;
         private Ordering order;
+        private int page;
+        private int perPage;
```

Comment:
I know it's an API limitation, but just looks odd doing it this way. Don't think there's anything we can do though

Reply:
`perPage` name is not descriptive name for me. `numberOfItemsPerPage` would be ideal for me but not sure if it's too long

Reply:
@pitiphong-p This just perspective from Omise API. BTW `numberOfItemsPerPage` sound a bit  too long. What about `itemsPerPage`?

Reply:
Yeah, maybe. What's the common convention for a Java library?

Reply:
That's also true